apps/web/src/views/Home/components/EcoSystemSection/index.tsx

```diff
@@ -377,16 +379,20 @@ const EcoSystemSection: React.FC = () => {
         <Flex
           style={{ gap: 32 }}
           flexDirection={isMobile || isMd ? 'column' : 'row'}
-          alignItems={isMobile || isMd ? undefined : 'center'}
+          alignItems={isMobile || isMd ? 'center' : 'center'}
```

Comment:
That means all-size `center`?

Reply:
yeah i should remove the boolean. cheers momota